### ISSUE TITLE
broken doc link

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4516]
|<img alt="ChrisW's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ChrisW reported|
|-|-|
|Trac ID|trac#4516|
|Type|defect|
|Created|2010-06-22 13:19:23Z|
|Branch|https://github.com/twisted/twisted/tree/broken-tutorial-link-4516|

http://twistedmatrix.com/documents/current/core/howto/tutorial/protocol.html#auto2 contains a link to http://twistedmatrix.com/documents/10.0.0/api/twisted.application.service.service.html

It should be to http://twistedmatrix.com/documents/10.0.0/api/twisted.application.service.Service.html

<details><summary>Searchable metadata</summary>

```
trac-id__4516 4516
type__defect defect
reporter__ChrisW ChrisW
priority__normal normal
milestone__ 
branch__branches_broken_tutorial_link_4516 branches/broken-tutorial-link-4516
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1277212763000000 1277212763000000
changetime__1277601139000000 1277601139000000
version__None None
owner__jesstess jesstess
cc__jesstess cc__thijs cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess removed owner|
|-|-|

[r29387] Fix broken links and spelling errors in the finger tutorial.

Also add missing code markup and adjust line wrapping.

refs [#4516](https://github.com/twisted/twisted/issues/4516)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess commented|
|-|-|

Thanks exarkun. For people like me who look up the branching revision on the ticket to know what to diff, that was:

r29381 A(branches/broken-tutorial-link-4516/): Branching to 'broken-tutorial-link-4516'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @jesstess|
|-|-|

Looks great, thanks!  Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_9"></a>|@jesstess set status to `closed`|
|-|-|

(In [29391]) Merge broken-tutorial-link-4516
Author: jesstess
Reviewer: exarkun
Fixes: [#4516](https://github.com/twisted/twisted/issues/4516)

Broken links and spelling errors in the finger tutorial are now fixed.
